src/repository/screens/repository.screen.js

```diff
@@ -275,44 +275,40 @@ class Repository extends Component {
           navigateBack
         >
           {initalRepository &&
-          !initalRepository.owner &&
-          isPendingRepository && (
+            !initalRepository.owner &&
```

Comment:
There is a lot of whitespaces changes in your PR that makes it really hard to read :(

(advice: I personally always perform a `git diff` on my changes before the commit and suppress any unwanted change in whitespaces. Helps me make sure my diff is minimalist and right to the PR point)

Reply:
This is probably because the code passes through the linter, and before this the code was not formalized according to the rules, so now we see so many changes. In general, nothing to do with this, except that do not create a new PR where all the code will be formatted according to the standards.